src/state/slices/opportunitiesSlice/types.ts

```diff
@@ -67,6 +67,8 @@ export type OpportunityMetadataBase = {
   underlyingAssetIds: AssetIdsTuple
   // The underlying amount of underlyingAssetId 0 and maybe 1 per 1 LP token, in base unit
   underlyingAssetRatiosBaseUnit: readonly string[]
```

Comment:
We should update the comment above, there can now be more than 0/1 underlying asset

Reply:
Updated in [b4210de](https://github.com/shapeshift/web/pull/7459/commits/b4210de685c8e4e4f44ee4f62b647f0eef04b04a)